src/typeInference/annotateTypeSwitchOperator.ts

```diff
@@ -84,11 +90,10 @@ function checkComparison(
 	// TODO: check this behaviours here as well (happens together with the TODO above)
 	// If there is no atomicType, we cannot compare the types and hence cannot infer the type and return item()*
 	if (!firstChild) {
-		const itemReturn = {
+		return {
 			type: ValueType.ITEM,
-			mult: SequenceMultiplicity.EXACTLY_ONE,
+			mult: SequenceMultiplicity.ZERO_OR_MORE,
```

Comment:
Is there a reason this changed from exactly_one to zero_or_more?

Reply:
Since we don't annotate it, shouldn't it be the broadest type possible?